wp-content/themes/amplify/functions.php

```diff
@@ -177,5 +189,10 @@ function save_lr_tag_meta( $term_id ) {
 		update_post_meta( $post_id, 'lr_more_link', sanitize_url( $_POST['lr_more_link'] ) );
 	}
 
+	// save the "Google Forms ID" meta if posible
+	if( isset( $_POST['lr_gforms_id'] ) && ! empty( $_POST['lr_gforms_id'] ) ) {
+		update_post_meta( $post_id, 'lr_gforms_id', sanitize_text_field( $_POST['lr_gforms_id'] ) );
+	}
+
```

Comment:
The present setup prevents the from ID from being unset. We need a case for when the form submission is empty, something like:

```
	if( isset( $_POST['lr_gforms_id'] ) && ! empty( $_POST['lr_gforms_id'] ) ) {
		update_post_meta( $post_id, 'lr_gforms_id', sanitize_text_field( $_POST['lr_gforms_id'] ) );
	} else if( isset( $_POST['lr_gforms_id'] ) && empty( $_POST['lr_gforms_id'] ) ) {
		delete_post_meta( $post_id, 'lr_gforms_id', sanitize_text_field( $_POST['lr_gforms_id'] ) );
	}
```

Or to go another step, as pseudocode:

```
$old = get_post_meta( $post_id, 'lr_gforms_id' );
if( isset( $_POST['lr_gforms_id'] ) && ! empty( $_POST['lr_gforms_id'] ) ) {
	update_post_meta( $post_id, 'lr_gforms_id', sanitize_text_field( $_POST['lr_gforms_id'] ) );
} else if( ! empty ( $old ) && isset( $_POST['lr_gforms_id'] ) && empty( $_POST['lr_gforms_id'] ) ) {
	delete_post_meta( $post_id, 'lr_gforms_id', sanitize_text_field( $_POST['lr_gforms_id'] ) );
}
```
so that we're not trying to delete metadata that doesn't exist

Reply:
Updated in https://github.com/INN/umbrella-inndev/pull/138/commits/d2299d689bec7efe50ca14e62e74681c1feee3b5